applications/FluidDynamicsApplication/custom_elements/d_vms_dem_coupled.cpp

```diff
@@ -81,6 +89,81 @@ Element::Pointer DVMSDEMCoupled<TElementData>::Create(
 ///////////////////////////////////////////////////////////////////////////////////////////////////
 // Input and output
 
+template <class TElementData>
+void DVMSDEMCoupled<TElementData>::Initialize(const ProcessInfo& rCurrentProcessInfo)
+{
+    // Base class does things with constitutive law here.
+    DVMS<TElementData>::Initialize(rCurrentProcessInfo);
+
+    const unsigned int number_of_gauss_points = this->GetGeometry().IntegrationPointsNumber(this->GetIntegrationMethod());
+
+    // The prediction is updated before each non-linear iteration:
+    // It is not stored in a restart and can be safely initialized.
+    mPredictedSubscaleVelocity.resize(number_of_gauss_points);
+    for (unsigned int g = 0; g < number_of_gauss_points; g++)
+        mPredictedSubscaleVelocity[g] = ZeroVector(Dim);
+
+    // The old velocity may be already defined (if restarting)
+    // and we want to keep the loaded values in that case.
+    if (mOldSubscaleVelocity.size() != number_of_gauss_points)
+    {
+        mOldSubscaleVelocity.resize(number_of_gauss_points);
+        for (unsigned int g = 0; g < number_of_gauss_points; g++)
+            mOldSubscaleVelocity[g] = ZeroVector(Dim);
+    }
+
+    #ifdef KRATOS_D_VMS_SUBSCALE_ERROR_INSTRUMENTATION
+    mSubscaleIterationError.resize(number_of_gauss_points);
+    mSubscaleIterationCount.resize(number_of_gauss_points);
+    #endif
```

Comment:
Which is the purpose of this? This compilation macro is neither defined nor used in the current master. It seems to me that it is a leftover from the very first implementation of the dynamic subscales element.

Reply:
Yes, it is a leftover.

Reply:
> Yes, it is a leftover. Same comment in `alternative_d_vms_dem_coupled.cpp`

So we can safely remove it?